lib/base/base.go

```diff
@@ -43,6 +43,7 @@ func Load(r *rt.Runtime) (rt.Value, func()) {
 		r.SetEnvGoFunc(env, "tonumber", tonumber, 2, false),
 		r.SetEnvGoFunc(env, "tostring", tostring, 1, false),
 		r.SetEnvGoFunc(env, "type", typeString, 1, false),
+		r.SetEnvGoFunc(env, "warn", warn, 0, true), // Added in Lua 5.4
```

Comment:
Does 0 nArgs mean variable number of arguments?

Reply:
The integer means a fixed number of arguments, the boolean is true if there are varargs as well